layouts/html.hbs

```diff
@@ -136,8 +140,20 @@
           iframeLoadedResolve();
         },
         onMessage: (message) => {
-          if (message.token) {
-            initAnswersJWT(message.token);
+          // Due to the async nature of the iframe resizer messaging, ensure
+          // that we only update the HH config if it is more recent than the last one
+          const messageContainsNewHHConfig = message.hitchhikerConfig
+             && message.timestamp >= hhConfigMetadata.lastReceived;
+          if (messageContainsNewHHConfig) {
+            window.HitchhikerConfig = {
+              ...window.HitchhikerConfig,
+              ...message.hitchhikerConfig
+            };
+            hhConfigMetadata.lastReceived = message.timestamp;
+          }
+          if (message.hitchhikerConfig?.token && !hhConfigMetadata.hasInitializedWithJWT) {
```

Comment:
Since Answers can't be initialized twice, I'm wondering if it would be beneficial to have a more general variable than `hasInitializedWithJWT`. Something like `hasAnswersInitialized`. We could then use that here and in other situations potentially.

Reply:
updated!